packages/cozy-client/examples/README.md

```diff
@@ -0,0 +1,20 @@
+This folder contains scripts to run cozy-client requests and a folder with `web` environment
+
+## How to run a script
+
+First start the stack.
+
+Example: to run the script file named `perfs`:
+
+```
+cd cozy-client/packages/cozy-client/examples
+node perfs.js
+```
+
+And then accept permissions / login.
+
+During the first login, it might crash with a `ERR_UNHANDLED_REJECTION`
```

Comment:
We do not know why this error?

Reply:
No, I would have mentioned the reason if so.

And I didn't take the time to investigate.